ariadne/executable_schema.py

```diff
@@ -37,8 +54,103 @@ def make_executable_schema(
 
     assert_valid_schema(schema)
 
+    for type_name, field_name, arg, _ in find_enum_values_in_schema(schema):
+        if is_invalid_enum_value(arg):
+            raise ValueError(
+                f"Value for type: <{arg.type}> is invalid. "
+                f"Check InputField/Arguments for <{field_name}> in <{type_name}> "
+                "(Undefined enum value)."
+            )
+
     return schema
 
 
 def join_type_defs(type_defs: List[str]) -> str:
     return "\n\n".join(t.strip() for t in type_defs)
+
+
+def find_enum_values_in_schema(
```

Comment:
Lets move this util somewhere else so `executable_schema.py` stays at current small and sweet size.

Reply:
Ok, done.